pkg/operator/controller/controller_cabundle_configmap_test.go

```diff
@@ -1,13 +1,12 @@
 package controller
 
 import (
-	"testing"
-
 	"github.com/google/go-cmp/cmp"
 	operatorv1 "github.com/openshift/api/operator/v1"
-
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
```

Comment:
The go packages should be first in the list.

Reply:
Ah good catch; Goland thwarted me again :-) I've fixed this locally and will push up the fix shortly.

Reply:
Fixed in https://github.com/openshift/cluster-dns-operator/pull/321/commits/b34b2f79a523ef62abd01b3dc7c3f326f839d957